extensions/ql-vscode/src/model-editor/model-editor-module.ts

```diff
@@ -188,6 +188,7 @@ export class ModelEditorModule extends DisposableObject {
         databaseItem: DatabaseItem,
       ) => {
         this.modelingStore.setSelectedMethod(databaseItem, method, usage);
+        await this.methodModelingPanel.show();
```

Comment:
This is the `codeQLModelEditor.jumpToUsageLocation` command which is called when clicking on items in the method usage panel. It doesn't trigger when clicking the "view" button in the main model editor. I assume we also want it to run then?

Reply:
I'm not sure we do tbh. I wanted to get opinions from EPD. This part hasn't exactly been spec'ed out. See internal issue for details.